lib/spack/spack/test/modules/__init__.py

```diff
@@ -0,0 +1,6 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+"""This file ensures pytest subdirs don't have name conflicts"""
```

Comment:
Remove this docstring too.    This string just ends up being confusing in the API docs.